apps/evm/src/__mocks__/models/tokens.ts

```diff
@@ -90,6 +91,14 @@ export const eth: Token = {
   asset: ethLogo,
 };
 
+export const weth: Token = {
+  address: '0x700868CAbb60e90d77B6588ce072d9859ec8E281',
+  decimals: 18,
+  symbol: 'WETH',
+  asset: wethLogo,
+  wrapsNative: true,
```

Comment:
Just an idea, but you could update the `wrapsNative` prop to `tokenWrapped`, so that the token being wrapped would be defined here directly. This could make this feature a bit more flexible and prevent having to look for the token being wrapped whenever we want to use it.